src/autocluster_k8s.erl

```diff
@@ -67,22 +67,34 @@ make_request() ->
       [{"Authorization", ["Bearer ", Token1]}],
       [{ssl, [{cacertfile, autocluster_config:get(k8s_cert_path)}]}]).
 
-%% @spec node_name(k8s_endpoint) -> list()
+%% @spec node_name(k8s_endpoint) -> list()  
 %% @doc Return a full rabbit node name, appending hostname suffix
 %% @end
 %%
 node_name(Address) ->
   autocluster_util:node_name(
     autocluster_util:as_string(Address) ++ autocluster_config:get(k8s_hostname_suffix)).
 
+
+%% @spec maybe_ready_address(k8s_subsets()) -> list()
+%% @doc Return a list of ready nodes
+%% SubSet can contain also "notReadyAddresses"  
+%% @end
+%%
+maybe_ready_address(Subset) ->
+  case proplists:get_value(<<"addresses">>, Subset) of 
+      undefined -> autocluster_log:info("No nodes ready yet!"), [];
```

Comment:
It is not a typo
We return an empty list, since the POD is still starting. Most likely is itself. 
here we are in this situation:
```
"subsets": [
    {
      "notReadyAddresses": [
        {
          "ip": "172.17.0.2",
```
Where we have **only** not ready nodes. 

Here I had two ways to handle this:

1.  Since the  StatefulSets can start the POD sequentially,  most likely is the first node.
2.  Consider `notReadyAddresses` as possible nodes to join and trust on `auto-delay` startup to join this node.